homework.md

```diff
@@ -4,110 +4,110 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+Red
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+HyperText Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS used to define style for web pages including layout, design and variations
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language commonly used in web development
```

Comment:
When you say that JavaScript is a programming language commonly used in web development you are **right** and at the same time **wrong**.

So why am I seeing this you may wonder? So you have to understand that JavaScript over the past few years has evolved so much ever since **2009** when **NodeJS** was introduced we could now build **CLI's and Web Services** using JavaScript and come **2015 when React Native** was introduced we could build Native Mobile applications and finally **2019 ElectronJS** was introduced and we could build Desktop application for all OS's.

So basically JavaScript is not only for Web Dev